src/core_modules/capture-core/rulesEngineActionsCreator/runRulesForSingleEvent.js

```diff
@@ -51,19 +38,34 @@ function getRulesEngineDataElementsAsObject(
 function getDataElements(program: Program) {
     let dataElements: Array<DataElement> = [];
 
-    if (program instanceof TrackerProgram) {
-        dataElements = getTrackerDataElements(program);
-    }
-
     if (program instanceof EventProgram) {
         dataElements = getEventDataElements(program);
     }
 
     return getRulesEngineDataElementsAsObject(dataElements);
 }
+
+// todo do we need to separate events by stage at this point when we dont have stages in the product?
```

Comment:
@JoakimSM can you please have a look on this comment?

Reply:
Agreed, it's redundant. Let's keep the byStage object though, so we don't break anything.

Reply:
agreed.